site-pages/sign-up/sign.css

```diff
@@ -0,0 +1,35 @@
+.signup-main{
+    display: flex;
+    margin-top: 30px;
+    margin-bottom: 30px;
```

Comment:
Can use margin:30px 0; instead of this

Reply:
Oh okay, shorthand property. Will change it.